### DISCUSSION TITLE
JXLS not showing hyperlink built as formula

### DISCUSSION THREAD

**Discussion by chienntgit** [jxlsteam/jxls#128] [Q&A] [answered]
I'm trying to generate excel using JXLS using latest java library
`    compile group: 'org.jxls', name: 'jxls', version: '2.10.0'
    compile group: 'org.jxls', name: 'jxls-poi', version: '2.10.0'
    compile group: 'org.jxls', name: 'jxls-jexcel', version: '1.0.9'`

I'm having problem generate hyperlinks in excel spreadsheet. This is how I'm building hyperlinks as formula in excel template
`=IF(K4<>"",HYPERLINK(K4,K4),"")`

This's working well in Mac OS excel spreadsheet as I can see hyperlinks in exported excel.

However it's NOT showing correctly when opening exported excel using Microsoft Excel 365
This's how it's showing when opening in Ms excel 365
`#VALUE!`
If I click on the row when press Enter then the link is displayed correctly

This is my template
[https://www.dropbox.com/s/3115hsnbduw1rei/order_summary_template.xls?dl=0](url)

and this is my report (Not showing well in Ms excel 365)
[https://www.dropbox.com/s/yahgr9msopjc2lo/order_summary_20-May-2021-1752.xls?dl=0](url)

Any response is much appreciated
Thanks

**Comment by SoltauFintel** [comment, marked as answer]
My recommendation is to create the templates with MS Excel 365 (Windows) from scratch.

**Reply by chienntgit** [reply]
Very good idea, thanks so much @SoltauFintel . I hadn't thought of that solution before. I'll try.

**Comment by chienntgit** [comment]
it helped and it's working well now @SoltauFintel 

Thanks a lot
